lib/interface/cli/helpers/cli-config.js

```diff
@@ -87,9 +88,23 @@ function propertyErrorHandler(e) {
     throw e;
 }
 
+function getSdkConfigForPagination() {
+    const conf = {};
+    const { HOSTNAME, CF_BUILD_ID } = process.env;
+
+    //  if cli runs inside of the codefresh pipeline need to create pagination sessionId according to the build id and container id
+    if (HOSTNAME && CF_BUILD_ID) {
```

Comment:
what will happen if 2 different steps will use the cli?
we should add here some identifier for the specific step - container id or something like that

Reply:
HOSTNAME is a container id where the step is executed, so it will create diff sessionIds for each step